versioned_docs/version-1.0.0/install-kubeslice/kubeslice-cli/topology-configuration.mdx

```diff
@@ -146,4 +176,21 @@ configuration:
       chart_name: #{The name of the Worker Chart}
       version: #{The version of the chart to use. Leave blank for latest version}
       values: #(Values to be passed as --set arguments to helm install)
+    
+    # The ui_chart, prometheus_chart, and image_pull_secret parameters are required for enterprise chart only
```

Comment:
Change `enterprise chart ` only to enterprise-demo (enterprise chart)  to be more clear.